feature_tests/current/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+PATH="$(pwd)":$PATH # simulating a custom `node`
```

Comment:
I'm not sure how it simulates a custom node

Reply:
but I'm merging this anyway, we can remove it later

Reply:
> I'm not sure how it simulates a custom node

You are absolutely right. Simulating custom `node` here isn't needed and this doesn't do that anyway 🤦